code/datums/outfits/eros.dm

```diff
@@ -152,3 +152,88 @@
 			uniform.attach_accessory(null, sbando)
 		else
 			qdel(sbando)
+
+
+/decl/hierarchy/outfit/eros_vox_carapace
+	name = "Vox Gear - Carapace (VOX ONLY)"
+	uniform =/obj/item/clothing/under/vox/vox_casual
+	l_ear = /obj/item/device/radio/headset/raider
+	glasses = /obj/item/clothing/glasses/thermal/plain/monocle
+	mask = /obj/item/clothing/mask/gas/swat/vox
+	suit = /obj/item/clothing/suit/space/vox/carapace
+	head = /obj/item/clothing/head/helmet/space/vox/carapace
+	back = /obj/item/weapon/storage/backpack/satchel_black
+	backpack_contents = list(/obj/item/weapon/storage/box/vox = 1)
+	shoes = /obj/item/clothing/shoes/magboots/vox
+	gloves = /obj/item/clothing/gloves/vox
+	suit_store = /obj/item/weapon/tank/nitrogen
+	r_hand = /obj/item/weapon/gun/launcher/alien/slugsling
+
+	id_slot = slot_wear_id
+	id_type = /obj/item/weapon/card/id/syndicate
+	id_pda_assignment = "Legitimate Merchant"
+
+/decl/hierarchy/outfit/eros_hap_sniper
```

Comment:
this is why we don't code at ass in the morning instead of sleeping
as you can see, this was a copypaste job